scenarios/bring-your-own-data/bringYourOwnData.md

```diff
@@ -0,0 +1,140 @@
+# Use Azure OpenAI On Your Data With openai-node Library
+
+## Objective
+
+Use openai-node Azure client to generate answers from your data using Azure On Your Data feature.
+
+This tutorial uses the following services:
+
+- Access to Azure OpenAI Service - you can apply for access [here](https://go.microsoft.com/fwlink/?linkid=2222006)
+- Access to Azure Search Service 
+
+## Time
+
+You should expect to spend 5-10 minutes running this sample. 
+
+## Before you begin
+
+### Set Environment Variables
+
+To start with let us create a config `.env` file with your project details. This file can be used in this sample or other samples to connect to your workspace. 
+
+```ts
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// You will need to set these environment variables or edit the following values
+// The endpoint you will use to access your Azure OpenAI instance
+const endpoint = process.env["ENDPOINT"] || "<endpoint>";
+// Your Azure OpenAI API key
+const azureApiKey = process.env["AZURE_API_KEY"] || "<api key>";
```

Comment:
Let's get rid of API key and promote only AAD. Also, let's let the client load the env var automatically. We can point to the user to set the `AZURE_OPENAI_ENDPOINT` var in their .env file.